tools/db_bench_tool.cc

```diff
@@ -235,6 +237,17 @@ DEFINE_bool(reverse_iterator, false,
 
 DEFINE_bool(use_uint64_comparator, false, "use Uint64 user comparator");
 
+DEFINE_bool(pin_slice, true, "use pinnable slice for point lookup");
+
+DEFINE_string(trace_file, "",
+              "If not empty string, use default tracer to trace the"
+              "workload with this as filename");
+
+DEFINE_string(replay_file, "",
```

Comment:
I am confused here. What is the difference between a trace_file and replay_file? Do we ever need two different files? Shouldn't it always be a trace file?